frontend/src/features/myinfo/utils/augmentWithMyInfoDisplayValue.ts

```diff
@@ -13,5 +13,10 @@ export const augmentWithMyInfoDisplayValue = (
   if (!isMyInfo(field)) return field
 
   const myInfoBlock = MAP_ATTR_TO_NAME[field.myInfo.attr]
-  return { ...field, fieldValue: myInfoBlock.previewValue }
+  return {
+    ...field,
+    fieldValue: myInfoBlock.previewValue,
+    // Leave default as false, same as in MyInfoData class
+    disabled: myInfoBlock.previewIsDisabled ?? false,
```

Comment:
seems like this may have a bug. The original `field.disabled` prop is ignored and overwritten by `false`.

```suggestion
    disabled: myInfoBlock.previewIsDisabled ?? field.disabled,
```

depending on the priority, hehe